ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template1.kt

```diff
@@ -70,52 +70,55 @@ private fun ColumnScope.Template1MainContent(state: PaywallState.Loaded) {
     val localizedConfig = state.selectedLocalization
     val colors = state.currentColors
 
-    Column(
-        modifier = Modifier
-            .fillMaxWidth()
-            .verticalScroll(rememberScrollState())
-            .fillMaxHeight(),
-        horizontalAlignment = Alignment.CenterHorizontally,
-        verticalArrangement = Arrangement.spacedBy(UIConstant.defaultVerticalSpacing, Alignment.CenterVertically),
-    ) {
-        HeaderImage(state.templateConfiguration.images.headerUri)
-
-        Spacer(modifier = Modifier.weight(1f))
-
-        Markdown(
-            text = localizedConfig.title,
-            style = MaterialTheme.typography.displaySmall,
-            fontWeight = FontWeight.Black,
-            textAlign = TextAlign.Center,
-            color = colors.text1,
-            modifier = Modifier
-                .padding(
-                    horizontal = UIConstant.defaultHorizontalPadding,
-                    vertical = UIConstant.defaultVerticalSpacing,
-                ),
-        )
-
-        Box(
+    if (state.isInFullScreenMode) {
+        Column(
             modifier = Modifier
-                .padding(horizontal = UIConstant.defaultHorizontalPadding),
+                .fillMaxWidth()
+                .verticalScroll(rememberScrollState())
+                .fillMaxHeight()
+                .weight(1f),
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.Center,
```

Comment:
Note that I removed the space between in the vertical arrangement. Instead, I'm relying on the `Spacer` inside the column. Those should stretch and contract depending on the available space. If it gets to too much content, it should become scrollable.